lib/persisted-model.js

```diff
@@ -600,6 +600,11 @@ module.exports = function(registry) {
     var typeName = PersistedModel.modelName;
     var options = PersistedModel.settings;
 
+    // This is just for LB 3.x
+    if (options.replaceOnPUT !== false || options.replaceOnPUT === undefined) {
```

Comment:
When `options.replaceOnPUT` is undefined, then `options.replaceOnPUT !== false` is truthy. Please simplify the condition to the following:

```js
if (options.replaceOnPUT !== false)
```